### PR TITLE
New Vizzu wasm url option used

### PR DESCRIPTION
**Pull request by simzer**
no need for "webpack-import-meta-loader" further on.

### CODE REVIEW

**Comment by simzer** [comment]
Tests done:
- freshly cloned repo builds and works correctly,
- new project built from scratch step by step based on the howto build and works.

File: docs/index.md [hunk 1]
```diff
@@ -287,12 +282,8 @@ Notice that we've added another `useRef()`. We need this to keep track of our `c
 
    - For Webpack (create-react-app, Next.js, Vite production mode):
 
-     1. Make sure you have `@open-wc/webpack-import-meta-loader` installed
+     1. Set the .wasm module location explicitely for Vizzu using `Vizzu.options({ wasmUrl })`
```
Comment (korompaiistvan) [inline docs/index.md]: **typo**

```suggestion
     1. Set the .wasm module location explicitly for Vizzu using `Vizzu.options({ wasmUrl })`
```

**Comment by korompaiistvan** [comment]
looks basically good to me, with one note: I feel that the line setting the Vizzu options should be moved out of the function body of `App()`, and into the top-level of the script.

I can't prepare a commit and test this now unfortunately

**Comment by simzer** [comment]
I just realized, that this last commit is incorrect, I moved the call out of useEffect() to the App() body and not into the top-level script. Why would the top level of the script be the appropriate place?

**Comment by simzer** [comment]
Anyway, I made the changes according to your instructions.

**Pull request merged**